src/Core/Grid/Definition/Factory/ImageTypeGridDefinitionFactory.php

```diff
@@ -0,0 +1,330 @@
+                            'placeholder' => $this->trans('Search Name', [], 'Admin.Actions'),
+                        ],
+                        'required' => false,
+                    ])
+                    ->setAssociatedColumn('name')
+            )
+            ->add(
+                (new Filter('width', TextType::class))
+                    ->setTypeOptions([
+                        'attr' => [
+                            'placeholder' => $this->trans('Search Width', [], 'Admin.Actions'),
+                        ],
+                        'required' => false,
+                    ])
+                    ->setAssociatedColumn('width')
+            )
+            ->add(
+                (new Filter('height', TextType::class))
+                    ->setTypeOptions([
+                        'attr' => [
+                            'placeholder' => $this->trans('Search Height', [], 'Admin.Actions'),
+                        ],
+                        'required' => false,
+                    ])
+                    ->setAssociatedColumn('height')
+            )
+            ->add(
+                (new Filter('products', YesAndNoChoiceType::class))
+                    ->setAssociatedColumn('products')
+            )
+            ->add(
+                (new Filter('categories', YesAndNoChoiceType::class))
+                    ->setAssociatedColumn('categories')
+            )
+            ->add(
+                (new Filter('manufacturers', YesAndNoChoiceType::class))
+                    ->setAssociatedColumn('manufacturers')
+            )
+            ->add(
+                (new Filter('suppliers', YesAndNoChoiceType::class))
+                    ->setAssociatedColumn('suppliers')
+            )
+            ->add(
+                (new Filter('stores', YesAndNoChoiceType::class))
+                    ->setAssociatedColumn('stores')
+            )
+            ->add(
+                (new Filter('actions', SearchAndResetType::class))
+                    ->setTypeOptions([
+                        'reset_route' => 'admin_common_reset_search_by_filter_id',
+                        'reset_route_params' => [
+                            'filterId' => self::GRID_ID,
+                        ],
+                        'redirect_route' => 'admin_image_settings_index',
+                    ])
+                    ->setAssociatedColumn('actions')
+            );
+
+        return $filters;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function getGridActions()
+    {
+        return (new GridActionCollection())
+            ->add(
+                (new LinkGridAction('import'))
+                    ->setName($this->trans('Import', [], 'Admin.Actions'))
+                    ->setIcon('cloud_upload')
+                    ->setOptions([
+                        'route' => 'admin_import',
+                        'route_params' => [
+                            'import_type' => 'customers',
```

Comment:
Just to be sure: is this correct? `import_type` = `customers` ?

Reply:
Oops!